README.md

```diff
@@ -5,3 +5,9 @@
 This is a simple python interface of RARBG.to.
 
 The api object will automatically fetch/refresh the token and take care rate control (1 request/2 seconds)
+
+# using
```

Comment:
Using `Installation` as session name would be nice.

Reply:
I don't understand. Can you elaborate?